lib/src/vs/oracle.rs

```diff
@@ -0,0 +1,167 @@
+//! System time based versionstamp.
```

Comment:
This module is used from nowhere for now.

However, this will be used as an alternative versionstamp producer for InMem, RocksDB, IndexedDB, and perhaps SpeeDB too.

For InMem and RocksDB, we will use `SysTimeCounter` as the runtime environment of a SurrealDB instance backed by either KVS should have a monotonic system clock, which `SysTimeCounter` relies on.

For IndexedDB, we will use `EpochCounter` which does not depend on the system clock.

Reply:
Yes, I think I could move ds/cl.rs Timestamp into this as well potentially; We may want to call it lib/src/time instead of vs. Not sure.
I need timestamp arithmetic that is synchronised across a cluster. So oracle timestamps T1, T2 and then `T1+5s<T2` for example. I know that's not how the hybrid logical clock works. But we need something centralised that is temporal instead of logical.